lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -92,6 +93,7 @@ def self.build_evm_event(event, passed_inputs = {})
       aevent.merge!("#{hash[:class].name}::#{hash[:name]}" => vmdb_object.id, "#{hash[:key]}_id".to_sym  => vmdb_object.id)
     end
 
+    aevent.merge!(:tenant_id => event_tenant(event).id) if event_tenant(event)
```

Comment:
@lfu @kbrock 
This tenant_id should also be passed into the call_automate or be set in the user.current_tenant